### PR TITLE
The whitespace parser only accepts space, fix #10

### LINKED ISSUE #10: The whitespace parser only accepts space
The comment claims that it parses any whitespace character, but the code only parses space:
https://github.com/gcanti/parser-ts/blob/01e0b935a3372fae43b556e512eeb5bb18896db3/src/char.ts#L51-L56

The comments in `lib/string` is also off in the same manner.

### PR DESCRIPTION
**Pull request by gcanti**
@danr thanks for the bug report, could you please review this fix?

### CODE REVIEW

File: test/char.ts [hunk 1]
```diff
@@ -45,6 +45,7 @@ describe('char', () => {
   it('space', () => {
     const parser = space
     eqEithers(parser.run(' '), createParseSuccess(' ', ''))
+    eqEithers(parser.run('\t'), createParseSuccess('\t', ''))
```
Comment (danr) [inline test/char.ts]: Add a test for `\n` too?

And perhaps test that `"\n\t "` is whitespace is `test/strings.ts`?

Comment (danr) [reply test/char.ts]: I like how you offload whitespace checking to a regex

Comment (gcanti) [reply test/char.ts]: @danr thanks, PR updated

**Comment by danr** [comment]
Looks great, go for it!

**Comment by gcanti** [comment]
Released

**Comment by danr** [comment]
Thank you!

**Pull request merged**